seed/views/organizations.py

```diff
@@ -195,6 +195,79 @@ def _save_fields(org, new_fields, old_fields, is_public=False):
 _log = logging.getLogger(__name__)
 
 
+class RulesSubSerializer(serializers.Serializer):
```

Comment:
Thanks @tallus.  I had thought about putting the serializers in a separate file, but to avoid complications with Swagger I left them in the same file as the views.  I'll make the change and test it out and if Swagger still understands how to find the serializers I'll push it up.